app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -31,7 +31,7 @@ $(function () {
         var beforeSlideChange = function(slider) {
             if ( !isSliderMasthead ) { return; }
 
-            $header.fadeOut();
```

Comment:
Switched this to fadeTo because fadeOut/fadeIn makes the header jump when the height is not fixed